src/Core/Installer.php

```diff
@@ -414,7 +412,6 @@ public function checkFunctions()
 			$returnVal = false;
 		}
 		$this->addCheck(DI::l10n()->t('IntlChar PHP module'), $status, true, $help);
```

Comment:
@MrPetovan that's the reason for the mock adaption. `$status` is only set in case the check fail. The other checks are passing the l10n text directly before the check. I think the best would be to refactor this whole method. But I think I won't be able to do that in the next days...